src/v2/models.py

```diff
@@ -240,13 +240,14 @@ def _normalize_ecosystem(ecosystem):
 
 
 class StackRecommendationResult(BaseModel):  # noqa: D101
-    _audit: 'Audit'
-    uuid: UUID
+    _audit: 'Audit' = None
+    uuid: UUID = None
     external_request_id: str
+    manifest_file_path: str = None
+    manifest_name: str = None
     registration_status: 'RegistrationStatus'
-    recommendation_status: 'RecommendationStatus'
+    recommendation_status: 'RecommendationStatus' = 'success'
     companion: List['RecommendedPackageData']
```

Comment:
Isnt this file generated, if that is the case I dont see any changes in the yaml.

Reply:
Yes, models.py has been generated using [datamodel-code-generator](https://pypi.org/project/datamodel-code-generator). Updated the spec to point to the default value for this field.